docs/workflows.rst

```diff
@@ -241,20 +241,19 @@ translators to translate into additional languages.
 Source strings reviews
 ----------------------
 
-With :ref:`project-source_review` enabled, the review process can be applied on
-the source strings. Once enabled, users can report issues in the source
-strings.  The actual process depends on whether you use bilingual or
-monolingual formats.
+With :ref:`project-source_review` enabled, the review process can be applied for
+source strings. Once enabled, users can report issues with source strings.
+The actual process depends on whether bilingual or monolingual formats are in use.
 
-For monolingual formats, the source string review behaves similarly as with
-:ref:`reviews` - once issue is reported on the source string, it is marked as
+For monolingual formats, source string review functions similarly to
+:ref:`reviews` — once an issue with a source string is reported, it is marked as
 :guilabel:`Needs editing`.
 
-The bilingual formats do not allow direct editing of the source strings (these
-are typically extracted directly from the source code). In this case
+The bilingual formats do not allow direct editing of source strings (these
+are typically extracted directly from the source code). In this case a
 :guilabel:`Source needs review` label is attached to strings reported by
 translators. You should review such strings and either edit them in the source
-or remove the label.
+code, or remove the label.
```

Comment:
?

Reply:
`source` is corresponding to `target`. It may refer to “source text”.

See https://en.m.wikipedia.org/wiki/Translation#Source_and_target_texts